terratest/utils/utils.go

```diff
@@ -268,3 +273,110 @@ func RunBusyBoxCommand(t *testing.T, options *k8s.KubectlOptions, dns string, co
 	}
 	return shell.RunCommandAndGetOutputE(t, cmd)
 }
+
+// K8gbRBAC apply or delete ad-hoc rbac to be able to read the nodes (should not clash with the "k8gb" cluster role and "k8gb" cluster role binding)
+func K8gbRBAC(t *testing.T, options *k8s.KubectlOptions, verb string) {
+	if verb != "apply" && verb != "delete" {
+		require.Failf(t, "you should call the K8gbRBAC with verb = apply | delete, it was: '%s'", verb)
+	}
+	rbacPath, err := filepath.Abs("../examples/rbac-geo-tags-read-nodes.yaml")
+	require.NoError(t, err)
+	cmd := shell.Command{
+		Command: "bash",
+		Args:    []string{"-c", fmt.Sprintf("cat %s | envsubst | kubectl %s -n %s -f -", rbacPath, verb, options.Namespace)},
+		Env:     map[string]string{
+			"namespace": options.Namespace,
```

Comment:
Do we controll k8gb namespace in tests?

Reply:
y, get's created & deleted [here](https://github.com/k8gb-io/k8gb/pull/823/files#diff-ff1e9de1e313a77323dc5ce09022081630f3f965935e1632db43b5193c38a43cR44). They will look like `k8gb-test-geo-tag-1fuk1x` where the last chunk is random

Reply:
We can throw away `envsubs` and use .Release.Namespace in original RBAC and ommit having separate rbac resource here.